Makefile.am

```diff
@@ -44,10 +44,10 @@ BUILT_SOURCES = src/builtin.inc src/version.h
 	$(AM_V_LEX) echo "NOT building lexer.c!"
 endif
 
-# Tell YACC (bison) autoconf macros that you want a header file created.
-# If the --warnings=all fails, you probably have an old version of bison
-# OSX ships an old bison, so update with homebrew or macports
-AM_YFLAGS = --warnings=all -d
+# Tell YACC (Bison) autoconf macros that you want a header file created.
+# If the --warnings=all fails, you probably have an old version of Bison
+# macOS ships an old Bison, so update with Homebrew or MacPorts.
+AM_YFLAGS = --warnings=all -Wno-yacc -d
```

Comment:
Wondering is we should at some point add some kind of dev docs as i always forget how set this, ex things like `YACC="$(brew --prefix)/opt/bison/bin/bison -y" ./configure ...` for brew/macOS

Reply:
I have `export PATH="$HOMEBREW_PREFIX/opt/bison/bin:$PATH"` in `.zprofile`. Maybe [wiki](https://github.com/jqlang/jq/wiki/Installation#from-github) can be improved (I don't like the symlink command).

Reply:
Ah yeap wiki could be good, want some place to dump it so it's easy to find and again

Reply:
I edited this page [like this](https://github.com/jqlang/jq/wiki/Installation/_compare/6ed0b47f03c857742f119c6caa959be08c5fd9fa...3e99f901ba8b8098f9da8e85636526c7b578c9bc), but you can edit further if you have better suggestion.

Reply:
👍